release_tester/arangodb/installers/deb.py

```diff
@@ -189,6 +190,32 @@ def un_install_package(self):
         except pexpect.exceptions.EOF:
             ascii_print(uninstall.before)
             sys.exit(1)
+        
+        if self.cfg.have_debug_package:
```

Comment:
this doesn't make sense here - init is invoked before anything else - why uninstall here?